lib/fastlane/plugin/appcenter/actions/appcenter_upload_action.rb

```diff
@@ -191,7 +192,24 @@ def self.run_release_upload(params)
             release = Helper::AppcenterHelper.update_release(api_token, owner_name, app_name, release_id, release_notes)
             Helper::AppcenterHelper.update_release_metadata(api_token, owner_name, app_name, release_id, dsa_signature)
 
-            destinations_array = destinations.split(',')
+            destinations_array = []
+            if destinations == '*'
```

Comment:
At this point we must make sure that the distribution type is `group`.

Reply:
Will add this :)